yellowbrick/features/base.py

```diff
@@ -75,11 +65,68 @@ def fit_transform_poof(self, X, y=None, **kwargs):
         return Xp
 
 
+class MultiFeatureVisualizer(FeatureVisualizer):
```

Comment:
So one point of feedback I've been getting is that the inheritance chain is too complicated and it's hard to reason about what methods are doing what. Not necessarily for this PR, but perhaps we could open an issue to create a series of Mixins that do the work we're describing for things like this.